### PR TITLE
Module/mutsig/1.0

### PR DESCRIPTION
**Pull request by Kdreval**
# Pull Request Checklists

**Important:** When opening a pull request, keep only the applicable checklist and delete all other sections.

### Required

- [X] I used the cookiecutter template and updated the placeholder rules.

- [X] The snakemake rules follow the [design guidelines](https://lcr-modules.readthedocs.io/en/latest/for_developers.html#module-rules). 

  - [X] All references to the `rules` object (_e.g._ for input files) are wrapped with `str()`.

- [X] Every rule in the module is either listed under `localrules` or has the `threads` and `resources` directives.

- [X] Input and output files are being symlinked into the `CFG["inputs"]` and `CFG["outputs"]` subdirectories, respectively.

- [ ] I grouped the input symlinking rule to the next job that uses the input files. 

- [X] I updated the final target rule (`*_all`) to include every output rule.

- [X] I explained important module design decisions in `CHANGELOG.md`.

- [X] I tested the module on real data for all supported `seq_type` values.

- [X] I updated the `default.yaml` configuration file to provide default values for each rule in the module snakefile.

- [X] I did not set any global wildcard constraints. Any/all wildcard constraints are set on a per-rule basis. 

- [X] I ensured that all symbolic links are relative and self-contained (_i.e._ do not point outside of the repository).

- [X] I replaced every value that should (or might need to) be updated in the default configuration file with `__UPDATE__`.

- [X] I recursively searched for all comments containing `TODO` to ensure none were left. For example:

  ```bash
  grep -r TODO modules/<module_name>/1.0
  ```

### If applicable

- [X] I added more granular output subdirectories.

- [ ] I added rules to the `reference_files` workflow to generate any new reference files.

- [ ] I added subdirectories with large intermediate files to the list of `scratch_subdirectories` in the `default.yaml` configuration file.

- [X] I updated the list of available wildcards for the input files in the `default.yaml` configuration file.

### CODE REVIEW

File: modules/mutsig/1.0/mutsig.smk [hunk 1]
```diff
@@ -0,0 +1,295 @@
...
+        ln -sf MutSig2CV/mutsig2cv/reference .
+            &&
+        touch configure.success
+        """)
+
+
+# Actual MutSig2CV run
+rule _mutsig_run:
+    input:
+        mcr = str(rules._mutsig_download_mcr.output.local_mcr),
```
Comment (lkhilton) [inline modules/mutsig/1.0/mutsig.smk]: I recommend wrapping all references to software installation outputs in `ancient()`.

**Comment by Kdreval** [comment]
I have confirmed it works in GAMBL settings :heavy_check_mark:

**Review by lkhilton — CHANGES_REQUESTED** [review]
One minor change requested!

**Comment by Kdreval** [comment]
I wrapped the installation inputs into `ancient()` and this is ready for re-review (or approval)

**Comment by Kdreval** [comment]
I have also wrapped the sample subsets file into ancient because I realised once you add a new subset to the table, it will want to re-run all previous subsets as well. This modification prevents from unnecessary re-runs, confirmed :heavy_check_mark:

**Review by lkhilton — APPROVED** [review]

**Pull request merged**